rfc9002.md

```diff
@@ -346,10 +346,10 @@ future RTT samples that are smaller than the new RTT to be included in
 smoothed_rtt.
 
 Endpoints SHOULD set the min_rtt to the newest RTT sample after persistent
-congestion is established. This is to allow a connection to reset its estimate
-of min_rtt and smoothed_rtt ({{smoothed-rtt}}) after a disruptive network event,
-and because it is possible that an increase in path delay resulted in persistent
-congestion being incorrectly declared.
+congestion is established. This helps when an increase in path delay
```

Comment:
Helps what?

The key is that it avoids repeatedly entering persistent congestion if the path RTT has genuinely increased.